.github/workflows/release.yml

```diff
@@ -30,7 +32,7 @@ jobs:
           RELEASE_NAME_PREFIX: "GIS Tools "
         with:
           args: |
-            ${{ env.ARC_TOOLBOX_ZIP }}
+            ${{ steps.get_version.outputs.ARC_FILE }}
```

Comment:
can this be on the same line as `args:`?

Reply:
It def could; if/when we go to multiple files (QGIS) we'll need to move it back to this